### PR TITLE
Stats: initial stats dashboard integration

### PR DESCRIPTION
**Pull request by dpaez**
Needs some work on the dashboard end, but the stats are reaching the dashboard.

### CODE REVIEW

File: packages/cli/src/pm2-async.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const pify = require('pify')
+const pm2 = require('pm2')
+
+module.exports.pm2Connect = pify(pm2.connect.bind(pm2))
```
Comment (dpaez) [inline packages/cli/src/pm2-async.js]: Why not to use native `util.promisify` instead of a new dep?

File: packages/cli/src/seeder-daemon [hunk 2]
```diff
@@ -8,6 +8,12 @@ const seederDaemon = async () => {
   const sdk = new SDK(config)
 
   await sdk.start()
+
+  process.on('message', function (packet, handler) {
```
Comment (dpaez) [inline packages/cli/src/seeder-daemon]: We need to removeListeners to not leak. Maybe on some destroy/close step?

**Review by dpaez — COMMENTED** [review]
Leave some comments, otherwise code looks good 👍

File: packages/cli/src/pm2-async.js (hunk 1 above)
Comment (estebanprimost) [reply packages/cli/src/pm2-async.js]: Solved. Thanks.

File: packages/cli/src/seeder-daemon (hunk 2 above)
Comment (estebanprimost) [reply packages/cli/src/seeder-daemon]: Done (on process exit stage)

**Review by estebanprimost — APPROVED** [review]

**Pull request merged**